### PR TITLE
fix mixers for tags

### PR DESCRIPTION
**Pull request by maximlopin**
- Add support for tags data type in `SklearnMixer`
- FIx schema of `depends_on_column` key in column config

### CODE REVIEW

**Review by George3d6 — CHANGES_REQUESTED** [review]
Please keep the booster mixer, since it's a good example mixer and we might want to use it in favor of sklearn for some tasks because training is quicker and in a lot of problems gradient boosters seems to be a very good speed/accuracy intersection

File: lightwood/mixers/base_mixer.py [hunk 1]
```diff
@@ -162,23 +162,22 @@ def calculate_accuracy(self, ds):
...
-                sample_weight = [1 for x in reals]
+                sample_weight = [1] * len(reals)
             else:
-                sample_weight = []
-                for val in reals:
-                    sample_weight.append(weight_map[val])
+                sample_weight = [weight_map[val] for val in reals]
+
+            if encoder is None:
+                raise ValueError('you must provide encoder to compue accuracy for tags data type')
```
Comment (paxcema) [inline lightwood/mixers/base_mixer.py:180]: typo: `compue` -> `compute`

File: lightwood/mixers/boost.py [hunk 2]
```diff
@@ -36,6 +42,22 @@ def _fit(self, train_ds, test_ds=None):
                 self.targets[target_col_name]['model'] = GradientBoostingClassifier(n_estimators=600)
                 self.targets[target_col_name]['model'].fit(X, Y, sample_weight=sample_weight)
 
+            elif self.targets[target_col_name]['type'] == COLUMN_DATA_TYPES.MULTIPLE_CATEGORICAL:
+                weight_map = self.targets[target_col_name]['weights']
+                if weight_map is None:
+                    sample_weight = [1] * len(Y)
+                else:
+                    sample_weight = [weight_map[val] for val in Y_train]
```
Comment (paxcema) [inline lightwood/mixers/boost.py:50]: Is `Y_train` defined in this method?

File: lightwood/mixers/sklearn_mixer.py [hunk 3]
```diff
@@ -92,28 +139,29 @@ def _fit(self, train_ds, test_ds):
...
-
                 if self.quantiles is not None:
                     self.targets[target_col_name]['quantile_models'] = {}
                     for i, quantile in enumerate(self.quantiles):
                         self.targets[target_col_name]['quantile_models'][i] = best_model_class(**best_model_kwargs)
 
             else:
                 self.targets[target_col_name]['model'] = None
-        
+                continue
```
Comment (paxcema) [inline lightwood/mixers/sklearn_mixer.py:149]: Seems to me like this `continue` is not needed?

**Review by paxcema — APPROVED** [review]

**Pull request merged**